server/activityItem/fetch.ts

```diff
@@ -152,12 +152,14 @@ const getActivityItemAssociationIds = (
 		thread,
 		user,
 	} = associationIds;
-	community.add(scope.communityId);
-	if ('pubId' in scope) {
-		pub.add(scope.pubId);
-	}
-	if ('collectionId' in scope) {
-		collection.add(scope.collectionId);
+	if (scope) {
+		community.add(scope.communityId);
+		if ('pubId' in scope) {
+			pub.add(scope.pubId);
+		}
+		if ('collectionId' in scope) {
+			collection.add(scope.collectionId);
+		}
```

Comment:
does this add activity notifications for users in some "scope"? is the scope a pub and/or collection